test/extensions/filters/http/cache/common.cc

```diff
@@ -0,0 +1,64 @@
+#include "test/extensions/filters/http/cache/common.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+std::ostream& operator<<(std::ostream& os, const RequestCacheControl& request_cache_control) {
+  std::string s = "{";
+  s += request_cache_control.must_validate_ ? "must_validate, " : "";
+  s += request_cache_control.no_store_ ? "no_store, " : "";
+  s += request_cache_control.no_transform_ ? "no_transform, " : "";
+  s += request_cache_control.only_if_cached_ ? "only_if_cached, " : "";
+
+  s += request_cache_control.max_age_.has_value()
+           ? "max-age=" + std::to_string(request_cache_control.max_age_.value().count()) + ", "
+           : "";
+  s += request_cache_control.min_fresh_.has_value()
+           ? "min-fresh=" + std::to_string(request_cache_control.min_fresh_.value().count()) + ", "
+           : "";
+  s += request_cache_control.max_stale_.has_value()
+           ? "max-stale=" + std::to_string(request_cache_control.max_stale_.value().count()) + ", "
+           : "";
+
```

Comment:
WYDT about one call to absl::StrCat for lines 9-23, rather than all these incremental appends?